dash/backend/src/gatekeeper-templates/Readme.md

```diff
@@ -1,5 +1,5 @@
-# Gatekeeper Libraries
+# gatekeeper Libraries

```

Comment:
https://github.com/open-policy-agent/gatekeeper

Looks like Gatekeeper can be capitalized (based upon what I'm seeing in their github)